nix/modules/scaffolding.nix

```diff
@@ -21,24 +20,26 @@
 
         buildInputs =
           (with pkgs; [
-            # TODO: remove sqlite package once https://github.com/holochain/holochain/pull/2248 is released
             openssl
+
+            # TODO: remove sqlite package once https://github.com/holochain/holochain/pull/2248 is released
             sqlite
```

Comment:
can be avoided by bumping holochain in the launcher to include these respectively:
* 0.1 https://github.com/holochain/holochain/commit/6de1fb217679bf0948769946652aebcea0df8876
* 0.2 https://github.com/holochain/holochain/commit/ca440c705eef2a2351a63d4da849ed4376c2222c